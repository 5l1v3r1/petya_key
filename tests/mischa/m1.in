b4QjQkFw2h8ua41xCUJRDYKTNWkiXGRuozLL2dWwFtFMEzku24i6D7phY2ht7QZXgqi1xxZNY1nhiJ4YLygF2GGgiZ908FA1
