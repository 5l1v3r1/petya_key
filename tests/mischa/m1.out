vW2ebtSboq7gBdUU
